extensions/aerial-photo/schema.json

```diff
@@ -90,6 +93,70 @@
         "aerial-photo:anomalies": {
           "title": "Image anomalies",
           "type": "string"
+        },
+        "summaries": {
+          "type": "object",
+          "required": ["aerial-photo:run", "aerial-photo:sequence_number"],
+          "properties": {
+            "aerial-photo:run": {
+              "type": "array",
+              "minLength": 1,
+              "minItems": 1,
+              "items": {
+                "type": "string"
+              }
+            },
+            "aerial-photo:sequence_number": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "lowest sequence number",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "highest sequence number",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:altitude": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "lowest altitude",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "highest altitude",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:scale": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "minimum scale",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "maximum scale",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:anomalies": {
```

Comment:
Should these fields be documented? `$comment` properties and readme docs would both be helpful.

Reply:
Do you mean documenting the summary of that property for the Collection as well as the Item Property which is already documented here: https://github.com/linz/stac/blob/master/extensions/aerial-photo/README.md ?

Reply:
I'm a fan of putting the documentation in the code and generating the readme, but since we don't have that in place yet we probably shouldn't duplicate it any more. Retracted.

Reply:
Yes, that would be nice.